Documentation/dev/libvirt-howto.md

```diff
@@ -85,6 +85,35 @@ iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
   -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
 ```
 
+If using `firewalld`, simply optain the name of the existing active zone which 
```

Comment:
Can you strip the trailing whitespace here (and in other places) to make Git happy:

```console
$ git log -2 --check --oneline origin/pr/284
c66b9c9 fixed a small typo
2ce46b2 adding details regarding using firewalld instead of the iptables
Documentation/dev/libvirt-howto.md:88: trailing whitespace.
+If using `firewalld`, simply optain the name of the existing active zone which 
Documentation/dev/libvirt-howto.md:99: trailing whitespace.
+unless otherwise specified. 
```